include/dt-bindings/adc/gd32f3x0.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2022 BrainCo Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ADC_GD32F3X0_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_ADC_GD32F3X0_H_
+
+#define RCU_ADCCK_IRC28M_DIV2	0
```

Comment:
I'd add `GD32_` prefix here.

Reply:
Just want to give a redefinition for the rcu_adc_clock_enum in here.

OK, I will change it.

Reply:
done.